.excludecoverage

```diff
@@ -0,0 +1,5 @@
+_mock.go
```

Comment:
can we add _matcher.go to this so we can start excluding all our custom matchers from coverage

Reply:
sure

Reply:
done